src/App.js

```diff
@@ -1,25 +1,27 @@
 import "./App.css";
 import Navbar from "./Components/Navbar/Navbar";
 
-import {Route,Routes} from "react-router-dom"
+import { Route, Routes } from "react-router-dom";
 import Home from "./Pages/Home/Home";
 import Mockman from "mockman-js";
 import { Login } from "./Pages/Authentication/Login";
 import Signup from "./Pages/Authentication/Signup";
 import NotFound from "./Pages/NotFound";
 import LikedVideo from "./Pages/LikedVideos/LikedVideo";
+import WatchLater from "./Pages/WatchLater/WatchLater";
```

Comment:
Ypu can import all of these files in index.js and import everything here in a single line. It will make your code look cleaner.

Reply:
fixed